lib/ardb.rb

```diff
@@ -27,6 +27,9 @@ def self.init(establish_connection = true)
     # setup AR
     ActiveRecord::Base.logger = self.config.logger
     self.adapter.connect_db if establish_connection
+  rescue InvalidDBFileError => exception
+    exception.set_backtrace(caller)
+    raise exception
```

Comment:
@jcredding also why bother rescuing and setting the custom backtrace on this at all?  The backtrace will never be displayed.